doc/ExpertOptions.md

```diff
@@ -1226,8 +1237,10 @@ If you want the default scatter chunk size for this workflow to be 100, but you
       "scatterDefaults": {
         "chunkSize": 100
       },
-      "sample_files.file": {
-        "chunkSize": 700
+      "scatters": {
+          "sample_files.file": {
+          "chunkSize": 700
+        }
```

Comment:
https://github.com/dnanexus/dxCompiler/blob/dd74c091f2ce2286c947dc334eb996fbdbc065b9/compiler/src/main/scala/dx/translator/Extras.scala#L383-L387

Reply:
Also in the test files:
https://github.com/dnanexus/dxCompiler/blob/9b0e5c7a8f366ea0f7e24190b1635c042453e3d9/compiler/src/test/resources/nested/extras/four_levels_extras1.json#L1-L14